juntagrico/templates/management_lists/assignments.html

```diff
@@ -53,15 +53,26 @@ <h3>
             {% for subscription in management_list %}
                 <tr>
                     <td>
-                        <a href="{% url 'admin:juntagrico_subscription_change' subscription.subscription.id %}">
-                            {{ subscription.subscription.recipients_names }}
-                        </a>
+                        {% spaceless %}
+                            <a href="{% url 'admin:juntagrico_member_change' subscription.subscription.primary_member.id %}">
+                                <strong>{{ subscription.subscription.primary_member }}</strong>
+                            </a>
+                        {% endspaceless %}
+                        {{ subscription.subscription.primary_member.get_phone }}
+                        <br>
```

Comment:
I'll add a checkbox and some javascript to collapse all co-members at once. That makes it easy to expand all in order to copy the list somewhere.

Reply:
we could do something like this
https://www.codegrepper.com/code-examples/javascript/copy+text+to+clipboard+button+html
so that users do not have to selct the stuff which might be cumbersome